keyboards/protokeeb/rev1/keyboard.json

```
@@ -0,0 +1,507 @@
+{
+    "usb": {
+        "pid": "0x1717",
+        "device_version": "1.0.0",
+        "force_nkro": false,
+        "max_power": 500,
+        "no_startup_check": false,
+        "polling_interval": 1,
+        "shared_endpoint": {
+            "keyboard": true,
+            "mouse": false
+        },
+        "suspend_wakeup_delay": 0,
+        "wait_for_enumeration": false
```

These are all the default behaviours.
```suggestion
        "device_version": "1.0.0"
```